src/fs.zig

```diff
@@ -288,6 +299,54 @@ pub fn findPath(
     return Error.Missing;
 }
 
+pub fn openStdoutFile(name: []const u8, comptime create: bool, clobber: clobberMode) !std.fs.File {
+    switch (clobber) {
+        .append, .forceClobber => {
+            if (openFile(name, create)) |file| {
+                return file;
+            }
+        },
+        .maybeClobber => {
+            // Check for noclobber
+            if (hsh.getState("set")) |ctx| {
```

Comment:
don't pull this from hsh here, I'm trying to reduce the dependency on hsh... and state is only for reading/writing the state to a file... using the ctx pointer here needlessly drops typesafety.